### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -267,23 +267,23 @@
...
-    <string name="error_fail_import">Failed to import file, check logcat.</string>
+    <string name="error_fail_import">Could not import file. Check logcat.</string>
     <string name="widget_placeholder">Click to restore %s</string>
-    <string name="add_widget_failed">Failed to add widget</string>
-    <string name="widget_placeholder_remove">Are you OK to remove the placeholder?</string>
-    <string name="choose_file_activity_not_found">Please install a File Manager.</string>
+    <string name="add_widget_failed">Could not add widget</string>
+    <string name="widget_placeholder_remove">Really remove the placeholder?</string>
+    <string name="choose_file_activity_not_found">Please install a file manager.</string>
     <string name="result_history_size">History max count</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:287]: "History max count" is not a good name.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -149,7 +149,7 @@
     <string name="screen_off_cache_clear_summary">Empty cache when screen turns off</string>
     <string name="memory_section">Memory</string>
     <string name="cache_half_apps">Small cache size</string>
-    <string name="cache_half_apps_summary">Make LRU cache size half the number of apps installed</string>
```
Comment (TBog) [inline app/src/main/res/values/strings.xml:152]: LRU cache - https://www.geeksforgeeks.org/lru-cache-implementation/

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -149,7 +149,7 @@
...
+    <string name="cache_half_apps_summary">Make URL cache size half the number of apps installed</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:152]:
```suggestion
    <string name="cache_half_apps_summary">Make LRU cache size half the number of apps installed</string>
```

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -223,42 +223,42 @@
...
     <string name="gesture_section">Gestures</string>
-    <string name="gesture_fling_down_left">Left fling down</string>
-    <string name="gesture_fling_down_left_summary">Fling downward starting on the left side of the screen</string>
-    <string name="gesture_fling_down_right">Right fling down</string>
-    <string name="gesture_fling_down_right_summary">Fling downward starting on the right side of the screen</string>
-    <string name="gesture_fling_up">Fling up</string>
-    <string name="gesture_fling_left">Fling left</string>
-    <string name="gesture_fling_right">Fling right</string>
+    <string name="gesture_fling_down_left">Fling from |<b>←</b>|, <b>↓</b></string>
+    <string name="gesture_fling_down_left_summary">Start from on the <b>→</b> side of the screen and fling <b>↓</b></string>
```
Comment (TBog) [inline app/src/main/res/values/strings.xml:242]: This should be starting from ← side

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (TBog) [reply app/src/main/res/values/strings.xml:287]: I agree. Use "History size" then?

**Pull request merged**